### DISCUSSION TITLE
I have been approved for the student benefits But still not getting any confirmation mail

### DISCUSSION THREAD

**Discussion by SuraiyaSultanaSweety** [community/community#58268] [GitHub Education]
### Hi everyone,

Hello everyone,
I have been granted the student benefits, but I have not received any email notification.
Approved Information: Congratulations! Your request for @JSuraiyaSultanaSweety has been approved, and your benefits will be available soon.
Additional Information: Hello, ***! You were last verified as a student 13 June 
![Screenshot_4](https://github.com/community/community/assets/136495078/b7b8bc90-d73a-455a-87bb-871bf039976b)
. There is no need for you to undergo verification again at this time. Please note that there may be a waiting period between verification and gaining access to academic benefits.

I have been waiting the whole day without receiving any response.
I would like to inquire about the duration of the waiting period after "approval." Will I receive an email from Github? If so, which email address will it be sent to—my primary email address, backup email address, or the email address used in my application?
To reiterate, my question pertains to whether my request has been approved rather than rejected.

**Comment by the-officialjosh** [comment]
I saw a similar issue a few days back. It seems you will have to be a bit patient with them as the feature will definitely be rolled out to you. Because most people with this complaint eventually get their benefits. Sorry for the delay anyways

**Comment by vaclavparma** [comment]
Same issue, some progress?

**Reply by the-officialjosh** [reply]
I will still give the same answer. The benefits will be given to you shortly

**Reply by HaomiaoPan** [reply]
Same issue, some progress?

**Comment by dishant-62** [comment]
I'm facing this also & my wait period is 3 days. Help me @the-officialjosh

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!

**Reply by tom1209-netizen** [reply]
I have the same problem, i have been verified on the 28 of August but still haven't received access to the student pack as of today

**Reply by mehulgupta1** [reply]
i am facing the same issue my request got approved but not getting the benifit

**Reply by watchmychops** [reply]
Same issue here, applied but keeps redirecting me to the page where I have to re-apply

**Reply by markwillcraft** [reply]
Same here huhu

**Reply by edilson14** [reply]
same problem here

**Reply by KanielDatz** [reply]
same problem here

**Reply by Habiba-09** [reply]
Same problem here XD

**Reply by games4life71** [reply]
same here

**Reply by 21dczhang** [reply]
same here
